lib/ttnt/tasks.rb

```diff
@@ -53,4 +50,6 @@ def install
   end
 end
 
-TTNT::TaskHelper.install_tasks
+# Define tasks when this file is required (this file is intended to be required from Rakefile)
+# ref: https://github.com/bundler/bundler//blob/master/lib/bundler/gem_tasks.rb
+TTNT::TaskHelper.define_tasks
```

Comment:
That's quite true. Since we cannot define tasks by only requiring a file (we need information from Rake::TestTask like test file names or patterns), your idea to define tasks in `TTNT::TestTask.new(t)` seems the best for me. Thank you for suggesting!

> Then you will have to handle the given name so we can have several tasks

That's great! I was wondering how I should handle when users define multiple `Rake::TestTask` instances with different names. I will open another pull request to handle it, thanks!